src/__tests__/jest/actions/authUser.test.js

```diff
@@ -0,0 +1,87 @@
+import configureMockStore from "redux-mock-store";
+import thunk from "redux-thunk";
+import moxios from "moxios";
+import expect from "expect";
+
+import {
+  START_FETCH,
+  SIGNUP_SUCCESS,
+  SIGNUP_ERROR
+} from "../../../redux/action_types";
+import authUser, {
+  signupSuccess,
+  signupError
+} from "../../../redux/actions/authUser";
+
+const middlewares = [thunk];
+const mockStore = configureMockStore(middlewares);
+const signupUrl = "http://127.0.0.1:8000/api/users/signup/";
+const validUser = {
+  username: "Tried and Tested",
+  email: "testeduser@gmail.com",
+  password: "12345678A"
+};
+const error = {
+  email: "please fill in the email"
+};
+
+describe("test signup_actions", () => {
+  beforeEach(() => {
+    moxios.install();
+  });
+  afterEach(() => moxios.uninstall());
+  it("should dispatch START_FETCH when the signup begins", done => {
```

Comment:
`done` at this point is only representing `()` as seen on line 38. Refactor to `it("should dispatch START_FETCH when the signup begins", () => ...` and delete line 38, no need to be having extra lines of code. Do the same for each test you have made use of `done()`